maat-court-data-api/src/test/java/gov/uk/courtdata/reporder/controller/RepOrderControllerTest.java

```diff
@@ -313,31 +329,85 @@ void givenInternalServerError_whenUpdateRepOrderIsInvoked_thenCorrectErrorRespon
+                .meansFullResult(null)
+                .meansFullStatus(null)
+                .meansAssessorName(MEANS_ASSESSOR_NAME_VALUE)
+                .dateMeansCreated(LocalDateTime.parse(DATE_MEANS_CREATED_VALUE))
+                .passportResult(PASSPORT_RESULT_VALUE)
+                .passportStatus(PASSPORT_STATUS_VALUE)
+                .passportAssessorName(PASSPORT_ASSESSOR_NAME_VALUE)
+                .datePassportCreated(LocalDateTime.parse(DATE_PASSPORT_CREATED_VALUE))
+                .fundingDecision(FUNDING_DECISION_VALUE)
+                .build();
+        when(repOrderService.findRepOrderStateByUsn(USN)).thenReturn(repOrderStateDTO);
+
+        mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + "/usn/" + USN))
+                .andExpect(status().isOk())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-                .andExpect(jsonPath("$", hasSize(0)));
+                .andExpect(jsonPath("$.usn").value(USN_VALUE))
+                .andExpect(jsonPath("$.maatRef").value(MAAT_REF_VALUE))
+                .andExpect(jsonPath("$.caseId").value(CASE_ID_VALUE))
+                .andExpect(jsonPath("$.caseType").value(CASE_TYPE_VALUE))
+                .andExpect(jsonPath("$.iojResult").value(IOJ_RESULT_VALUE))
+                .andExpect(jsonPath("$.iojAssesorName.fullName").value(IOJ_ASSESSOR_FULL_NAME))
+                .andExpect(jsonPath("$.iojAssesorName.userName").value(IOJ_ASSESSOR_USERNAME))
+                .andExpect(jsonPath("$.dateAppCreated").value(DATE_APP_CREATED_VALUE))
+                .andExpect(jsonPath("$.iojReason").doesNotExist())
+                .andExpect(jsonPath("$.meansInitResult").value(MEANS_INIT_RESULT_VALUE))
+                .andExpect(jsonPath("$.meansInitStatus").value(MEANS_INIT_STATUS_VALUE))
+                .andExpect(jsonPath("$.meansFullResult").doesNotExist())
+                .andExpect(jsonPath("$.meansFullStatus").doesNotExist())
+                .andExpect(jsonPath("$.meansAssessorName").value(MEANS_ASSESSOR_NAME_VALUE))
+                .andExpect(jsonPath("$.dateMeansCreated").value(DATE_MEANS_CREATED_VALUE))
+                .andExpect(jsonPath("$.passportResult").value(PASSPORT_RESULT_VALUE))
+                .andExpect(jsonPath("$.passportStatus").value(PASSPORT_STATUS_VALUE))
+                .andExpect(jsonPath("$.passportAssessorName").value(PASSPORT_ASSESSOR_NAME_VALUE))
+                .andExpect(jsonPath("$.datePassportCreated").value(DATE_PASSPORT_CREATED_VALUE))
+                .andExpect(jsonPath("$.fundingDecision").value(FUNDING_DECISION_VALUE));
+    }
+
+    @Test
+    void givenValidUsn_whenFindRepOrderByUsnIsInvokedAndFindsNoRepOrder_thenCorrectResponseIsReturned() throws Exception {
+        when(repOrderService.findRepOrderStateByUsn(USN)).thenReturn(null);
+
+        mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + "/usn/" + USN))
+                .andExpect(status().isOk());
```

Comment:
On lines 336-337 above you have:

```
mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL)
                        .param("usn", String.valueOf(USN)))
```

Is it possible to pass the param here in the same way for consistency?